diff -Naur python/src/cpipython.cpp python-new/src/cpipython.cpp
--- python/src/cpipython.cpp	2007-11-23 15:55:15.000000000 -0600
+++ python-new/src/cpipython.cpp	2009-07-29 15:00:15.382231568 -0500
@@ -1306,8 +1306,9 @@
 	if (topic && strlen(topic) < 1024)
 	{
 		cpiPython::me->server->mC.hub_topic = topic;
-		string msg;
-		cpiPython::me->server->mP.Create_HubName( msg, cpiPython::me->server->mC.hub_name, topic );
+		string msg, sTopic;
+		sTopic = topic;
+		cpiPython::me->server->mP.Create_HubName( msg, cpiPython::me->server->mC.hub_name, sTopic);
 		cpiPython::me->server->mUserList.SendToAll(msg, eUC_NORMUSER, eUC_MASTER);
 	}
 	return cpiPython::lib_pack("s", strdup(cpiPython::me->server->mC.hub_topic.c_str()));
